FutureNorthants/northants-design-system
# Add Google Translate component

- Add a Google Translate component, [inspiration from here](https://codesandbox.io/p/sandbox/google-translate-in-react-js-qzdjj?file=%2Fsrc%2FApp.js)
- Add option to display the Google Translate component in the [footer](https://deploy-preview-485--northants-design-system.netlify.app/?path=/story/library-structure-footer--example-footer-with-translate)

## ToDo
- [x] Get a list of languages for the dropdown and update dropdown.

## Testing
- Checkout this branch and run `npm run dev`
- View the Components -> Google Translate examples and test out the Google translate dropdown to see the same text translated
- View the Structure -> Footer examples and see the Example Footer With Google Translate
